docs/source/overview.rst

```diff
@@ -1,12 +1,12 @@
 Overview
 ========
 
-django-declarative-apis is a framework built on top of Django aimed at teams implementing RESTful APis. It provides
+django-declarative-apis is a framework built on top of Django aimed at teams implementing RESTful APIs. It provides
```

Comment:
I wonder why Sphinx isn't rendering the bullets for the list below?

![image](https://user-images.githubusercontent.com/624123/124183654-25cf2900-da7e-11eb-8f64-2adfb90e9e6a.png)

Reply:
Yeah, it was working and then suddenly stopped rendering the bullets. It still works with numbered list though.

Reply:
Upgrading the `sphinx_rtd_theme` dev dependency seems to fix it for me.